test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -1200,6 +1200,41 @@ public async Task SchemaCompareOpenScmpProjectToProjectRequest()
             await CreateAndOpenScmp(SchemaCompareEndpointType.Project, SchemaCompareEndpointType.Project);
         }
 
+        /// <summary>
+        /// Verify folder structure gets stored in scmp file for project endpoint
+        /// </summary>
+        [Test]
+        public async Task VerifyExtractTargetInScmpFile()
+        {
+            SqlTestDb sourceDb = await SqlTestDb.CreateNewAsync(TestServerType.OnPrem, false, null, SourceScript, "SchemaCompareOpenScmpSource");
+            SqlTestDb targetDb = await SqlTestDb.CreateNewAsync(TestServerType.OnPrem, false, null, TargetScript, "SchemaCompareOpenScmpTarget");
```

Comment:
why does a target db need to be created if the target in the schema compare is a project?

Reply:
For now, I was just following the pattern used in the test file to create project. Indeed a db is not required to have a project but this is what other tests are doing currently. May be a separate work item should be created to fix it at all the places?